docs/src/implemented-proposals/tower-bft.md

```diff
@@ -16,25 +16,42 @@ For brevity this design assumes that a single voter with a stake is deployed as
 
 The Solana cluster generates a source of time via a Verifiable Delay Function we are calling [Proof of History](../cluster/synchronization.md).
 
-Proof of History is used to create a deterministic round robin schedule for all the active leaders. At any given time only 1 leader, which can be computed from the ledger itself, can propose a fork. For more details, see [fork generation](../cluster/fork-generation.md) and [leader rotation](../cluster/leader-rotation.md).
+Proof of History is used to create a deterministic round robin schedule for all the active leaders. 
+
+The unit of time is called a "slot". Each slot has a designated leader that can
+produce a block `B`. The `slot` of block `B` is designated `slot(B)`. Leaders
+do not necessarily need to generate a block for their slot, in which case there
+may not be blocks for some slots.
+
+For more details, see [fork generation](../cluster/fork-generation.md) and [leader rotation](../cluster/leader-rotation.md).
+
+## Votes
+
+Validators communicate which fork they think is the heaviest through votes.
+Each vote `v` is signed by the validator that produces it, and is of the form `(i, B)`, where `i` is the validaotr producing the vote and `B` is the block being voted for.
```

Comment:
nit: validaotr -> validator
Hmm, I think we have more than that in votes now?

Reply:
This is a high level description of the algorithm and so I simplified the vote to the minimum necessary to make the algorithm understandable.

If we want an exact documentation of the code, I think that should be done elsewhere

Reply:
Will update this to reflect that we include the whole tower once that feature is stabliized